bin/mvn-search

```diff
@@ -1,4 +1,24 @@
 #!/usr/bin/env node
+const yargs = require('yargs');
+
+const args = yargs
+    .scriptName('mvn-search')
+    .usage('$0 <query-string> [args]', 'search for maven dependency', (yargs) => {
+        yargs.positional('query-string', {
+            describe: 'the dependency you search for',
+            type: 'string'
+        })
+    })
+    .option('format', {
+        alias: 'f',
+        description: 'Define in which format to print dependency. (kotlin, KotlinGroovy, Maven)',
```

Comment:
shoudn't this be:

```suggestion
        description: 'Define in which format to print dependency. (gradle, gradlegroovy, maven)',
```

?

Reply:
It is case insensitive anyway :)

Reply:
Sure but it’s not „Kotlin“ and „KotlinGroovy“ it’s „Gradle“ and „GradleGroovy“, no?

Reply:
🤦

Reply:
Thanks for pointing that out :)